iroh-cli/src/commands.rs

```diff
@@ -85,6 +85,16 @@ pub(crate) enum Commands {
     /// For general configuration options see <https://iroh.computer/docs/reference/config>.
     Console,
 
+    /// Shutdown the running node.
+    Shutdown {
+        /// Shutdown mode.
+        ///
+        /// Hard shutdown will immediately terminate the process, soft shutdown will wait
+        /// for all connections to close.
```

Comment:
```suggestion
        /// Using force will immediately terminate the process, otherwise shutdown
        /// will wait for all connections to close.
```